completions/Makefile.am

```diff
@@ -1,1575 +1,22 @@
-		_sentry-cli \
-		_starship
-	$(ss) mfiutil \
-		mrsasutil
-	$(ss) mplayer \
-		gmplayer kplayer mencoder mplayer2
-	$(ss) mutt \
-		muttng neomutt
-	$(ss) _nox \
-		_airflow \
-		_ansible \
-		_ansible-config \
-		_ansible-console \
-		_ansible-doc \
-		_ansible-galaxy \
-		_ansible-inventory \
-		_ansible-playbook \
-		_ansible-pull \
-		_ansible-vault \
-		_conda \
-		_cz \
-		_waydroid
-	$(ss) nslookup \
-		host
-	$(ss) p4 \
-		g4
-	$(ss) perl \
-		perldoc
-	$(ss) pine \
-		alpine
-	$(ss) ping \
-		ping4 ping6
-	$(ss) _pip \
-		_pip3
-	$(ss) _pipenv \
-		_black _blackd _flask _httpx
-	$(ss) pkg-config \
-		pkgconf
-if BSD
-	$(ss) pkg_delete \
-		pkg_deinstall pkg_info
-endif
-	$(ss) pgrep \
-		pkill
-	$(ss) pm-hibernate \
-		pm-suspend pm-suspend-hybrid
-	$(ss) psql \
-		createdb createuser dropdb dropuser
-	$(ss) postmap \
-		postalias
-	$(ss) povray \
-		spovray xpovray
-	$(ss) puppet \
-		filebucket puppetca puppetd puppetdoc puppetmasterd puppetqd \
-		puppetrun ralsh
-	$(ss) pytest \
-		py.test py.test-2 py.test-3 pytest-2 pytest-3
-	$(ss) pydoc \
-		pydoc3 pydoc3.13 pydoc3.14 pydoc3.15
-	$(ss) pylint \
-		pylint-2 pylint-3
-	$(ss) python \
-		micropython pypy pypy3 pyston pyston3 python2 python2.7 \
-		python3 python3.3  python3.4 python3.5 python3.6 python3.7 \
-		python3.8 python3.9 python3.10 python3.11 python3.12 \
-		python3.13 python3.14 python3.15
-	$(ss) pyvenv \
-		pyvenv-3.4 pyvenv-3.5 pyvenv-3.6 pyvenv-3.7 pyvenv-3.8 \
-		pyvenv-3.9 pyvenv-3.10 pyvenv-3.11 pyvenv-3.12 pyvenv-3.13 \
-		pyvenv-3.14 pyvenv-3.15
-	$(ss) qdbus \
-		dcop
-	$(ss) qemu \
-		qemu-kvm qemu-system-i386 qemu-system-x86_64
-	$(ss) quota \
-		edquota quotacheck quotaoff quotaon repquota setquota
-	$(ss) rcs \
-		ci co rcsdiff rlog
-	$(ss) rpm \
-		rpmbuild rpmbuild-md5
-	$(ss) rpm2tgz \
-		rpm2targz rpm2txz
-	$(ss) _ruff \
-		_mado \
-		_uv
-	$(ss) smbclient \
-		smbcacls smbcquotas smbget smbpasswd smbtar smbtree
-	$(ss) sbcl \
-		sbcl-mt
-	$(ss) sha256sum \
-		b2sum cksum md5sum shasum sha1sum sha224sum sha384sum sha512sum
-	$(ss) _sops \
-		_kata-runtime _todoist
-	$(ss) ssh \
-		autossh scp sftp sidedoor slogin
-	$(ss) sudo \
-		sudo-rs sudo.ws sudoedit
-	$(ss) tar \
-		bsdtar gtar star
-	$(ss) tracepath \
-		tracepath6
-	$(ss) update-alternatives \
-		alternatives
-	$(ss) _vault \
-		_bao \
-		_consul \
-		_fx \
-		_nomad \
-		_packer \
-		_terraform \
-		_tofu \
-		_xc
-	$(ss) vipw \
-		vigr
-	$(ss) vncviewer \
-		tightvncviewer xvnc4viewer
-	$(ss) _wasmer \
-		_wasmer-headless
-	$(ss) wine \
-		wine-development wine-stable \
-		wine64 wine64-development wine64-stable
-	$(ss) wodim \
-		cdrecord
-	$(ss) xz \
-		pxz
-	$(ss) ypmatch \
-		ypcat
 .PHONY: symlinks
 
 SETUP_SYMLINKS = $(srcdir)/../setup-symlinks.sh
 
 all-local: ss = $(SETUP_SYMLINKS) .
 all-local: symlinks
 
-install-data-hook: ss = $(SETUP_SYMLINKS) $(DESTDIR)$(bashcompdir)
+install-data-hook: ss = $(SETUP_SYMLINKS) $(DESTDIR)$(bashcompdir_fallback)
 install-data-hook: symlinks
 
```

Comment:
We should probably still create the directory when installing, so external completions can be placed here easily
Maybe using something [like this](https://stackoverflow.com/a/8531548)?

Same for `helpers/` if users are expected to put their own stuff there.